packages/core/test/combinator/multicast-test.js

```diff
@@ -0,0 +1,49 @@
+import { describe, it } from 'mocha'
+import { assert, eq } from '@briancavalier/assert'
+import { spy } from 'sinon'
+
+import { now } from '../../src/source/now'
+import { map, tap } from '../../src/combinator/transform'
+import { multicast } from '../../src/combinator/multicast'
+import { runEffects } from '../../src/runEffects'
+import { ticks } from '../helper/testEnv'
+
+describe('multicast', () => {
```

Comment:
Over in the mostjs/multicast repo, there are [tests for MulticastSource and MulticastDisposable](https://github.com/mostjs/multicast/tree/master/test).  I think it's important to port those as well.  You'll just need to export MulticastSource and MulticastDisposable for the purposes of unit testing them.  I'd be fine with copying those tests into `multicast-test.js`.

It kinda raises the question of whether we should split up multicast.js and multicast-test.js, but I'm ok with *not* splitting them up in this PR.

Also, on the topic of exports, currently [`@most/multicast` exports MulticastSource](https://github.com/mostjs/multicast/blob/master/src/index.js#L10).  It seems like we may also want to export it from `@most/core`.  Thoughts?